sdk/identity/Azure.Identity/samples/OtherCredentialSamples.md

```diff
@@ -16,3 +16,35 @@ var client = new SecretClient(new Uri("https://keyvault-name.vault.azure.net/"),
 ```
 
 ***Note:*** This credential is **not** included in the `DefaultAzureCredential` chain.
+
+
+# OnBehalfOfCredential with Managed Identity FIC Example
+
+This example demonstrates the use of the `OnBehalfOfCredential` to authenticate the `SecretClient` using a managed identity as the client assertion. More information about the On Behalf Of Flow can be found [here](https://learn.microsoft.com/entra/identity-platform/v2-oauth2-on-behalf-of-flow).
```

Comment:
nit: For a second, I thought `SecretClient` was an "example" client. Consider specifying?

```suggestion
This example demonstrates the use of the `OnBehalfOfCredential` to authenticate the KeyVault `SecretClient` using a managed identity as the client assertion. More information about the On Behalf Of Flow can be found [here](https://learn.microsoft.com/entra/identity-platform/v2-oauth2-on-behalf-of-flow).
```